### ISSUE TITLE
Add `wiby.json` support

### ISSUE THREAD

**Issue by BethGriggs** [pkgjs/wiby#8]
- [ ] Support the --file argument
- [ ] Reads a wiby.json (initially based on the draft structure)

**Comment by dominykas** [comment]
>  Support the --file argument

Proposal:

```bash
wiby test
```

Is equivalent to:

```bash
wiby test --config ./wiby.json
```

Is equivalent to:


```bash
wiby test -c ./wiby.json
```

---

- Choosing `--config` over `--file`, partially because that's what eslint uses, but also `file` is kind of vague?
- If the file does not exist - prints help
- Should also be able to fall back to `./wiby.js`

---

Happy to start implementing this, if nobody else is working on that yet?

**Comment by dominykas** [comment]
- Basics implemented in #45 
- Followup issues created, see #52
